processor/batchprocessor/README.md

```diff
@@ -15,8 +15,11 @@ any data drops such as sampling.
 Please refer to [config.go](./config.go) for the config spec.
 
 The following configuration options can be modified:
-- `send_batch_size` (default = 8192): Number of spans or metrics after which a
-batch will be sent regardless of the timeout.
+- `send_batch_size` (default = 8192): Number of spans, metric data points, or log
+records in metrics after which a batch will be sent regardless of the timeout.
```

Comment:
`records in metrics` they are not only in metrics. Same everywhere you refer to metrics

Reply:
I agree that `records in metric` was a wrong description.
But in the rest of the README it refers only to batching metrics and the non-even split.

Reply:
@bogdandrutu friendly ping

Reply:
removed this part of the Readme as it is no longer valid.